src/lib/ui/pressable/component.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Pressable, PressableProps, StyleSheet, StyleProp, ViewStyle } from 'react-native';
+
+export interface AppPressableProps extends PressableProps {
+  style?: StyleProp<ViewStyle>;
+}
+
+export function AppPressable({ children, style, ...props }: AppPressableProps) {
+  return (
+    <Pressable style={({ pressed }) => StyleSheet.flatten([{ opacity: pressed ? 0.4 : 1 }, style])} {...props}>
```

Comment:
To make component more flexible, we could allow customizing `pressedOpacity`:

```suggestion
    <Pressable style={({ pressed }) => StyleSheet.flatten([{ opacity: pressed ? pressedOpacity : 1 }, style])} {...props}>
```